src/main/java/uk/gov/hmcts/cmc/claimstore/models/Claim.java

```diff
@@ -126,35 +142,34 @@ public LocalDateTime getCountyCourtJudgmentRequestedAt() {
 
     @Override
     @SuppressWarnings("squid:S1067") // Its generated code for equals sonar
-    public boolean equals(Object other) {
-        if (this == other) {
-            return true;
-        }
-        if (other == null || getClass() != other.getClass()) {
-            return false;
-        }
-        Claim otherClaim = (Claim) other;
-        return Objects.equals(id, otherClaim.id)
-            && Objects.equals(submitterId, otherClaim.submitterId)
-            && Objects.equals(submitterEmail, otherClaim.submitterEmail)
-            && Objects.equals(letterHolderId, otherClaim.letterHolderId)
-            && Objects.equals(defendantId, otherClaim.defendantId)
-            && Objects.equals(externalId, otherClaim.externalId)
-            && Objects.equals(referenceNumber, otherClaim.referenceNumber)
-            && Objects.equals(claimData, otherClaim.claimData)
-            && Objects.equals(createdAt, otherClaim.createdAt)
-            && Objects.equals(issuedOn, otherClaim.issuedOn)
-            && Objects.equals(responseDeadline, otherClaim.responseDeadline)
-            && Objects.equals(respondedAt, otherClaim.respondedAt)
-            && Objects.equals(countyCourtJudgment, otherClaim.countyCourtJudgment)
-            && Objects.equals(countyCourtJudgmentRequestedAt, otherClaim.countyCourtJudgmentRequestedAt);
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS00122)